src/extensions/default/bramble/main.js

```diff
@@ -120,6 +121,13 @@ define(function (require, exports, module) {
 
         // We're all done loading and can pass startup state info back to the host app.
         RemoteEvents.loaded();
+
+        // Show Tutorial Pane by default if `tutorial.html` exists at project load
```

Comment:
You would grab the value in here: https://github.com/mozilla/brackets/blob/master/src/bramble/client/main.js#L276-L301 and then you'd fire your event with the value you get.